src/pages/en/core-concepts/astro-components.md

```diff
@@ -236,20 +248,25 @@ const { name } = Astro.props
   <slot />  <!-- children will go here -->
   <Footer />
 </div>
+```
+
+
+#### Named Slots
 
-// src/components/Person.astro
+You can pass multiple pieces of content to your Astro component, and place them precisely, using named slots. 
```

Comment:
```suggestion
You can precisely pass multiple pieces of content directly into your Astro component, by *naming* the slots. 
```

Reply:
I prefer the existing wording here — “passing precisely” seems confusing to me. But maybe the emphasis is helpful: `using **named slots**`?

Reply:
Still got to finish reviewing this, 😬